### DISCUSSION TITLE
Modernize the splashscreen

### DISCUSSION THREAD

**Discussion by FJBDev** [mytourbook/mytourbook#1215] [Q&A]
This is the current MT splash screen

![image](https://github.com/mytourbook/mytourbook/assets/25235665/8978caa2-977d-4b0a-a7b8-bd4a8f84ff3c)

Recently, I saw the new LibreOffice 7 splash screen and I was very inspired by the icons in 3D that they designed, it is very modern and "dynamic"

<img src="https://linuxsecurity.com/images/articles/vendors-products/libreoffice71-1-scaled.jpg" width="400">

Is someone skilled in either [Inkscape](https://inkscape.org/), [Blender](https://www.blender.org/), [3DSMax ](https://www.autodesk.com/products/3ds-max/overview)... to "modernize" the MT splash image ?

I have some ideas of what could look like the new splash but I don't have the skills

**Comment by wolfgang-ch** [comment]
A helmet is very essential to be also displayed in the splash screen, without it I probably would have not survived my bicycle accident 18 years ago

**Comment by stvzldn** [comment]
Maybe have an AI do the artwork?

**Reply by FJBDev** [reply]
Is that something AI can do ? Do you know much about it ?

**Reply by stvzldn** [reply]
I had not used it, but based on your response I tried an AI image creator at https://www.iamfy.co/studio with the following prompt:
"_Bicyclist on bicycle and hiker with rucksack. In background are a sports watch and a graph with elevation trace._"
This yielded the attached images.  It seemed to combine the bicyclist and the hiker in a single figure.  Likely would take some work adjusting the prompt for more suitable image results.  Other option is to use DALL-E directly from https://labs.openai.com/ which requires buying credits.

![Image1](https://github.com/mytourbook/mytourbook/assets/106929478/37198cf1-a8ef-438b-9488-ccd2af3567ff)
![Image2](https://github.com/mytourbook/mytourbook/assets/106929478/c3587720-0c24-40a7-94cf-831697b31479)

**Reply by FJBDev** [reply]
Interesting, that is something new to me.
I may have a better idea and some news to share soon, stay tuned... 😄

**Reply by FJBDev** [reply]
After several weeks of exchanging with [Barbara Tostes](https://www.barbara.blog.br/blog/)(the creator of the beautiful LibreOffice logos), she was very kind to agree to redesign the MyTourbook splash screen 😀
And what she designed is beautiful !

Here is the first rendering of her 3d blend file and her history behind the track.

Please let me know what you think and if you have skills in Blender.
Otherwise i think it would be a great first update for the 24.1 MT splash logo
![splash_002](https://github.com/mytourbook/mytourbook/assets/25235665/f2a5a066-3e9f-4878-a61c-679f5f049113)
![splash_001](https://github.com/mytourbook/mytourbook/assets/25235665/321c8d35-1461-4721-8ef4-33716987693a)

> I believe that with the .blend file you can feel free to create animations or opening videos/tutorials. They can also write the name of the software later on the rendered images (or include it in the 3D rendering). Show it to people on the forum and we'll change/adapt it... If anyone knows how to use Blender, feel free to change anything you want!

>See the two images I rendered (by pressing F12) - one with details on the helmet and sneakers.

>I left the reference images either...  I created a famous bike trail here in Brazil (my cousin did it! He took 7 days to complete it, it's called "Volta das Transições" - I send an image to view). Then just switch to another trail that’s relevant to you!

>Happy new year!
![Renato-Samel-Quadro-Volta-Transicoes](https://github.com/mytourbook/mytourbook/assets/25235665/755daa4b-63e0-4888-8d84-543572dc6bd8)

**Reply by FJBDev** [reply]
> A helmet is very essential to be also displayed in the splash screen, without it I probably would have not survived my bicycle accident 18 years ago

She made sure to put a helmet on 😀

**Reply by wolfgang-ch** [reply]
I like the helmet and sneaker design but the background mountains and helmet+sneaker+ring are a bit too dark.

The old ring had a gradient, the top is red (for the head) bottom is bright orange. There was also a drak ring which should display a camera lens for the photos which can be displayed. Instead of this, there could be displayed a smaller image on the track.

Here is a comparison of new and old

![image](https://github.com/mytourbook/mytourbook/assets/1283445/c647a1ad-d583-4a92-8651-1e6bee67a7cd)

.



.

![image](https://github.com/mytourbook/mytourbook/assets/1283445/c57330e2-8f7b-43f6-a007-c3306293df12)

**Reply by FJBDev** [reply]
>I like the helmet and sneaker design but the background mountains and helmet+sneaker+ring are a bit too dark.

I forgot to attach the blend file she created. Maybe you can play with them to change the colors.

The comparison with old and new is pretty good already 👌 

[MyTourBook_logo_splash_ideas.zip](https://github.com/mytourbook/mytourbook/files/13787301/MyTourBook_logo_splash_ideas.zip)

**Reply by wolfgang-ch** [reply]
> Maybe you can play with them to change the colors.

I'll try but blender is a very complex tool. 

When I tried to load the Blender file into my current Blender 3.3 installation, it crashed but with the current Blender 4.x, I can open the file.

**Comment by bybinhabr** [comment]
Hi, guys! I'm so glad that forum members are enjoying it. To a gradient effect in logo, we can light the scene better. I only left 1 light and they are made of solid materials (you can add texture or improve the colors, of course). You can also edit the rendered image later. We can add more details to the scene, but this may "take away" from the name of the software. You can change whatever you want, if there is anyone else who uses Blender on the forum. 

The track is just an example model, you can change it for another one or add more background graphics (graphics that your software generates), edit it in an image editor, merge the images, etc.

I'll try to make some changes soon... send more "printscreen" (images/graphics) of your software, please!

Best regards,

Barbara

**Reply by wolfgang-ch** [reply]
Hi Barbara

First of all many thanks for your art work for the new splash screen 🎉 

I agree with you to put not too much into the splash screen but I would like to see some (2?) photos like in the 2.5D map which is also one of the main features.

![image](https://github.com/mytourbook/mytourbook/assets/1283445/3c72b097-9831-4423-9053-e1ccd93a0ea1)


I'm a totally beginner for blender, I could change colors but was not able to set a gradient effect to the logo. So it would be very helpful when you could do this.

The only thing which I already could do is to set an additional camera position 😄 

![image](https://github.com/mytourbook/mytourbook/assets/1283445/c30ae094-f232-4c21-84bc-6de0dd40e517)

You can find screenshots for the app here https://mytourbook.sourceforge.io/mytourbook/index.php/screenshots

These are the images which you can try to show it on the track

![DSC00710](https://github.com/mytourbook/mytourbook/assets/1283445/315bdd4d-aec8-4d60-b343-51dcfdd15146)
![P1000050](https://github.com/mytourbook/mytourbook/assets/1283445/fdc47fb0-77ab-4970-ae2a-c3a09dd64509)

I already did some 3D work in the 2.5D map but this was no art work

https://github.com/mytourbook/mytourbook/issues/516#issuecomment-1426033808

https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/show-tours/2-5d-map/25d-model-player

https://user-images.githubusercontent.com/1283445/218138293-289543c9-7668-44b6-95c6-486fbf55aeea.mp4

**Comment by FJBDev** [comment]
> Hi, guys! I'm so glad that forum members are enjoying it. To a gradient effect in logo, we can light the scene better. I only left 1 light and they are made of solid materials (you can add texture or improve the colors, of course). You can also edit the rendered image later. We can add more details to the scene, but this may "take away" from the name of the software. You can change whatever you want, if there is anyone else who uses Blender on the forum.
> 

Thank you again @bybinhabr  for your contribution. I am sure that @wolfgang-ch will add you to the list of precious [contributors](https://mytourbook.sourceforge.io/mytourbook/index.php/contributors)
Your splashscreen is really impressive!

However, like Wolfgang said, I also don't have any skills in Blender, which looks rather like a complex tool to use with a pretty steep learning curve. But maybe some of the MT users have Blender skills that they could use to refine the splashscreen ? anyone @mytourbook/active-users ??


> The track is just an example model, you can change it for another one or add more background graphics (graphics that your software generates), edit it in an image editor, merge the images, etc.
> 
> I'll try to make some changes soon... send more "printscreen" (images/graphics) of your software, please!

Thanks for your upcoming changes!
Attached are the logos that I think are the main ones in MT
![MT-ImportantLogos](https://github.com/mytourbook/mytourbook/assets/25235665/46f71574-96f5-4abf-9dfe-48145c8d38cc)

**Comment by stvzldn** [comment]
Splash looks great on the install of 24.10.0 - SUPER

**Reply by FJBDev** [reply]
Glad you like it @stvzldn 👍

**Comment by GurVJ** [comment]
Nice new Splash screen !!
